app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+    source_file = parts[1]
+    destination_file = parts[2]
+    if source_file == destination_file:
+        return
+    with (open(source_file, "r") as file_in,
+          open(destination_file, "w") as file_out):
+        for line in file_in:
+            file_out.write(line)
```

Comment:
Avoid using loop here
```suggestion
        file_out.write(file_in.read())
```